### PR TITLE
Create page in AMPLIFY Unified Catalog “manage-catalog-assets”

### PR DESCRIPTION
**Pull request by jnamburi**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by andreamussap** [comment]
@jnamburi @georgiana-caras '_Edit catalog asset icon_' and _Edit catalog asset description_ sections are too trivial IMHO, so not needed. Don't you think?

My only suggestion would be to change the Alt text of the **pen** icon from '**Edit**' to something more meaningful '**Edit catalog asset Details**' (Can you ask this to UX team?)

**Comment by andreamussap** [comment]
https://apicentral.axway.com/catalog/explore
I've clicked an asset but the EDIT button wasn't available.
Why I don't have rights to edit it?
Will this happen to users, will they see assets that they can't EDIT the description?
What's the requirement? Shouldn't you document this in the doc?

File: content/en/docs/catalog/manage-catalog-assets.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+1. Click on the **pen** icon.
+2. Select **Manual** from the **Subscription approval** drop-down.
+3. Save changes.
+
+{{< alert title="Note" color="info" >}}Only users that are assigned the Amplify Central Admin role can perform this task.{{< /alert >}}
+
+## Add or remove tags
+
+Tags are keywords that can be attached to catalog assets and can be used to search catalog assets by tags. Unified Catalog allows both lower case and upper case tags, and the `_ - () \ []` special characters.
```
Comment (andreamussap) [inline content/en/docs/catalog/manage-catalog-assets.md:56]: @jnamburi "both lower case and upper case tags" --> does this mean that the search is case sensitive? If I create a tag  **MyCatalog** and search for **mycatalog** I won't see any results?

**Comment by georgiana-caras** [comment]
> I've clicked an asset but the EDIT button wasn't available.
> Why I don't have rights to edit it?
> Will this happen to users, will they see assets that they can't EDIT the description?
> What's the requirement? Shouldn't you document this in the doc?
To edit the Overview doc, you have to be assigned either an API Central Admin role, or an API Developer role. The API Developer can edit the assets in their team context.

**Pull request merged**